### PR TITLE
Bank UI component

### PR DESCRIPTION
**Pull request by denalimarsh**
This PR implements a Bank UI component in the substrate-ui browser-based application. It uses metamask to interface with a local Ethereum chain, allowing users to deposit Ethereum and ERC20 tokens to the locally deployed Bank contract.

Set up information is located in the readme. As the setup process is manual, double-check that your locally deployed contracts have the same ABI as the substrate-ui contracts do.

### CODE REVIEW

File: packages/app-polkadot-ethereum-bridge/src/Bank.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  useEffect(() => {
+    const fetchAccounts = async (web3: Web3) => {
+      const accs = await web3.eth.getAccounts();
+      let defaultAcc = accs[0];
+      web3.eth.defaultAccount = defaultAcc
+      setDefaultAccount(defaultAcc);
+    };
+
+    fetchAccounts(web3);
+  }, []);
```
Comment (philipstanislaus) [inline packages/app-polkadot-ethereum-bridge/src/Bank.tsx:47]: Should we not add `web3` here?

File: packages/app-polkadot-ethereum-bridge/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# @app-polkadot-ethereum-bridge
+
+A simple application that supports the deposit of Ethereum and ERC20 tokens to a Bank contract.
+
+## Setup
+
+### Start the local substrate chain
+```bash
+# In Snowfork/polkadot-ethereum-bridge/parachain
+./target/release/artemis-node --dev
```
[the remark this answers is not in the document]
Comment (denalimarsh) [reply packages/app-polkadot-ethereum-bridge/README.md:13]: https://github.com/Snowfork/substrate-ui/pull/1/commits/f0e6f3f97ed7f16fa7b08299a9d48011e966c78e

File: packages/app-polkadot-ethereum-bridge/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+### Start a local Ethereum chain
+```bash
+# In Snowfork/polkadot-ethereum-bridge/ethereum
+truffle develop
```
[the remark this answers is not in the document]
Comment (denalimarsh) [reply packages/app-polkadot-ethereum-bridge/README.md:21]: https://github.com/Snowfork/substrate-ui/pull/1/commits/f0e6f3f97ed7f16fa7b08299a9d48011e966c78e

File: packages/app-polkadot-ethereum-bridge/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+nvm use 10.18
+
+rm ./yarn.lock 
+
+yarn
+```
+
+```bash
+# In Snowfork/substrate-ui/packages/app-polkadot-ethereum-bridge
+npm install
```
[the remark this answers is not in the document]
Comment (denalimarsh) [reply packages/app-polkadot-ethereum-bridge/README.md]: Removed in https://github.com/Snowfork/substrate-ui/pull/1/commits/f0e6f3f97ed7f16fa7b08299a9d48011e966c78e

File: packages/app-polkadot-ethereum-bridge/src/Bank.tsx (hunk 1 above)
Comment (denalimarsh) [reply packages/app-polkadot-ethereum-bridge/src/Bank.tsx:47]: Removed in https://github.com/Snowfork/substrate-ui/pull/1/commits/28b438d1d43f33f3017cc5a54101636996e7553a - will remove the state variables from function params as well in the next iteration.

**Comment by denalimarsh** [comment]
Screen Shot 2020-07-15 at 1 10 18 AM

ETH deposit component is displayed immediately. After approving tokens, the ERC20 deposit component is displayed.

**Pull request merged**